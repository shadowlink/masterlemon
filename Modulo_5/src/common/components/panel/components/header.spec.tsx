import * as React from 'react';
import { shallow } from 'enzyme';
import { Header } from './header';

describe('common/components/panel/header tests', () => {
  it('should render as expected when passing required properties', () => {
    // Arrange
    const props = {
      title: 'test title',
    };

    // Act
    const component = shallow(
      <Header
        {...props}
      />
    );

    // Assert
    expect(component).toMatchSnapshot();
  });
});
