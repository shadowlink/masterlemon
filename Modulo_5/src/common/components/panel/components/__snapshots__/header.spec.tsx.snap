// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/components/panel/header tests should render as expected when passing required properties 1`] = `
<div
  className="card-header header"
>
  <h3
    className="panel-title"
  >
    test title
  </h3>
</div>
`;
